src/plugins/remote-cache/index.ts

```diff
@@ -58,6 +53,7 @@ async function turboRemoteCache(
       clientEmail: instance.config.GCS_CLIENT_EMAIL,
       privateKey: instance.config.GCS_PRIVATE_KEY,
       projectId: instance.config.GCS_PROJECT_ID,
+      useTmp: instance.config.STORAGE_PATH_USE_TMP_FOLDER ?? true,
```

Comment:
Since we have the default on `STORAGE_PATH_USE_TMP_FOLDER: Type.Optional(Type.Boolean({ default: true }))` we don't need it here.